src/routes/api/bookings/createBooking.ts

```diff
@@ -3,12 +3,14 @@ import timeBlockModel from '../../../models/timeBlock.model';
 import sectionModel from '../../../models/section.model';
 import Room from '../../../models/room.model';
 import { DateTime } from 'luxon';
+import booking from '../../../models/booking.model';
```

Comment:
Swap `booking` with `Booking` so that it's clear that it's a module (and not a variable)